pkgs/tools/package-management/nix/default.nix

```diff
@@ -6,13 +6,14 @@
 , storeDir ? "/nix/store"
 , stateDir ? "/nix/var"
 , confDir ? "/etc"
+, fromGit ? false
 }:
 
 let
 
   sh = busybox-sandbox-shell;
 
-  common = { name, suffix ? "", src, fromGit ? false }: stdenv.mkDerivation rec {
+  common = { name, suffix ? "", src, fromGit ? fromGit }: stdenv.mkDerivation rec {
```

Comment:
This creates an infinite recursion during eval. One of the two `fromGit` variables needs renaming.